backend/testfiles/execution/stdlib/result.dark

```diff
@@ -101,6 +101,117 @@ PACKAGE.Darklang.Stdlib.Result.map2_v0
 
 
 
+PACKAGE.Darklang.Stdlib.Result.map3_v0
+  (PACKAGE.Darklang.Stdlib.Result.Result.Error "error1")
+  (PACKAGE.Darklang.Stdlib.Result.Result.Error "error2")
+  (PACKAGE.Darklang.Stdlib.Result.Result.Error "error3")
+  (fun (a, b, c) -> a - b - c) = PACKAGE.Darklang.Stdlib.Result.Result.Error "error1"
```

Comment:
`(a, b, c)` here is a tuple, so it confuses me that this works (since the rest seems to be using non-tuples, which is correct).

`(fun a b c -> a - b - c)` is the non-tuple syntax.